modules/040-node-manager/hooks/machineclass_checksum_assign.go

```diff
@@ -165,10 +165,18 @@ func readChecksumTemplate(cloudType string) ([]byte, error) {
 }
 
 func getChecksumTemplatePath(cloudType string) string {
+	const modulesDirCE = "/deckhouse/modules"
+	const modulesDirEE = "/deckhouse/ee/modules"
+	const modulesDirFE = "/deckhouse/ee/fe/modules"
```

Comment:
Why we need this const?

Reply:
I've inlined this constants in https://github.com/deckhouse/deckhouse/pull/3740/commits/5fef301bb025fabb248f6d4fab7fc49136fd8485.